plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx

```diff
@@ -1,19 +1,25 @@
 import React, { useState } from 'react';
 import IconButton from '@mui/material/IconButton';
 import Box from '@mui/material/Box';
-import { InfoCard } from '@backstage/core-components';
+import { InfoCard, InfoCardVariants} from '@backstage/core-components';
 import { FetchComponent } from '../FetchComponent';
 import OpenInNewIcon from '@mui/icons-material/OpenInNew';
 import { ReadmeDialog } from '../ReadmeDialog/ReadmeDialog';
 
-export const ReadmeCard = () => {
+type ReadmeCardProps = {
```

Comment:
You should also export these props to enable better usability. By doing this, you will also need update of the api reports. You can read more here: https://github.com/AxisCommunications/backstage-plugins/blob/main/CONTRIBUTING.md#changesets